common/scala/src/main/scala/whisk/core/entity/WhiskAction.scala

```diff
@@ -150,6 +150,51 @@ case class WhiskAction(
     }
 
     def toJson = WhiskAction.serdes.write(this).asJsObject
+
+    def toExecutableWhiskAction = ExecutableWhiskAction(namespace, name, exec.asInstanceOf[CodeExec[_]], limits, version, rev)
```

Comment:
Should this return a try or option and defer to the caller to handle the case of a failing cast?